### PR TITLE
chore: 🤖 update starters.yml to include mui starter

### PR DESCRIPTION
**Pull request by siriwatknp**
## Description

add gatsby-mui-layout-starter to starters.yml

### Documentation

None

## Related Issues

None

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -1,3 +1,19 @@
+- url: https://mui-treasury.com/layout/clones/reactjs?bgColor=b6c0d4
+  repo: https://github.com/mui-treasury/gatsby-mui-layout-starter
+  description: Supercharge your next project with Mui Treasury Layout, built on top of Material-UI
+  tags:
+    - Styling:CSS-in-JS
+    - Layout
+    - Material-UI
```
Comment (gillkyle) [inline docs/starters.yml:7]:
```suggestion
    - Styling:Material
```

This is what the tag in our whitelist uses, so this should be changed to match 🙂

**Comment by LekoArts** [comment]
Hey @siriwatknp 👋 Can you address the review comments? Thanks!

File: docs/starters.yml [hunk 2]
```diff
@@ -6781,3 +6781,19 @@
     - Landing Page
     - Google Analytics
     - PaperCSS style
+- url: https://mui-treasury.com/layout/clones/reactjs?bgColor=b6c0d4
+  repo: https://github.com/mui-treasury/gatsby-mui-layout-starter
+  description: Supercharge your next project with Mui Treasury Layout, built on top of Material-UI
+  tags:
+    - Layout
```
Comment (gillkyle) [inline docs/starters.yml:6788]:
```suggestion
```

This isn't a tag we have in our allow list, so ought to be removed 🙂

File: docs/starters.yml [hunk 3]
```diff
@@ -6781,3 +6781,19 @@
...
+    - Styling:CSS-in-JS
+    - Styling:Material-UI
```
Comment (gillkyle) [inline docs/starters.yml:6790]:
```suggestion
    - Styling:Material
```

This one needs to match the tag in the allow list exactly.

**Pull request merged**